### PR TITLE
Case-control studies

### PR DESCRIPTION
**Pull request by iaindillingham**
This PR introduces a new section on case-control studies to accompany the functionality introduced by opensafely-core/cohort-extractor#694.

I've linked to this section from _OpenSAFELY best practice_ in the navigation.

### CODE REVIEW

File: docs/case-control-studies.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    run: cohortextractor:latest generate_cohort --study-definition study_definition_potential_controls
+    outputs:
+      highly_sensitive:
+        cohort: output/input_potential_controls.csv
+```
+
+## Match the cases to the potential controls
+
+In this step, we will use the [OpenSAFELY matching library](https://github.com/opensafely-core/matching#readme) in [a scripted action](actions-scripts.md) to match the cases to the potential controls.
+We will name this scripted action *match.py*.
```
Comment (alexwalkerepi) [inline docs/case-control-studies.md:71]: Could add "You can also use your own custom action to match cases to controls."

Comment (iaindillingham) [reply docs/case-control-studies.md:76]: Thanks @alexwalkerepi 🙂 Done in 4a2b6ba.

File: docs/case-control-studies.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
+There are five steps to undertaking a [case-control study](https://en.wikipedia.org/wiki/Case%E2%80%93control_study) with OpenSAFELY:
```
Comment (johntaz) [inline docs/case-control-studies.md:5]: Thanks @iaindillingham - just had a read through the documentation and it looks great. My only suggestion would be to add '(or matched cohort study)' after 'case-control study'.

Comment (iaindillingham) [reply docs/case-control-studies.md]: Thanks @johntaz. Done in b2d318f.

**Review by alexwalkerepi — APPROVED** [review]

**Pull request merged**